smalldata_tools/DetObject.py

```diff
@@ -298,6 +298,13 @@ def processSums(self):
                 except:
                     pass
 
+            #loop over kwargs & look for them in the key
```

Comment:
... down here, you seem to be looking for something else?  Not the word "dropped" surely?

Reply:
See the other comment: while I've only used it with 'dropped', this should allow users to make different event selections - one of the CXI groups had asked for that (though this may not be enough for them, I forgot the exact use case). I also think that for the liquid jet experiments, they sometimes look at only laser-off shots or combine off & on to extract the beam center as physics contains in the 'plain' sum messes with the shape.